projects/components/src/tabs/content/tab-group.component.ts

```diff
@@ -10,7 +10,16 @@ import { TabComponent } from './tab/tab.component';
       <mat-tab-group animationDuration="0ms" disableRipple (selectedTabChange)="this.activeTabIndex = $event.index">
         <mat-tab *ngFor="let tab of this.tabs; index as i">
           <ng-template mat-tab-label>
-            <div class="tab-label">{{ tab.label }}</div>
+            <div class="tab-label">
+              {{ tab.label }}
+              <div
+                *ngIf="tab.subText && tab.subText !== undefined"
```

Comment:
Just `tab.subText` this condition should be sufficient right?

Reply:
Yep. That should work.

Reply:
fixed!!